src/css/reviewPage.css

```diff
@@ -0,0 +1,18 @@
+/* Review Page */
+.review-page__page-heading__text {
+  color: var(--color-primary-darker);
```

Comment:
Might be a good idea to use `.vads-u-color--primary-darker ` here, that way we can remove this entire CSS rule.